Rakefile

```diff
@@ -58,7 +58,7 @@ end
 # The error count will need to be adjusted here until such time as all grammars are 100% error free.
 desc "Check that compiling the grammars doesn't introduce any new unexpected errors"
 task :check_grammars do
-  expected_error_count = 35  # This count should only ever go down. If it goes up, there's a new issue that needs to be addressed before updating the grammar.
+  expected_error_count = 37  # This count should only ever go down. If it goes up, there's a new issue that needs to be addressed before updating the grammar.
```

Comment:
I don't see any errors for missing grammars, but I did get a 2 extra errors, here's the full output:

```
- [ ] repository `https://bitbucket.org/Clams/sublimesystemverilog/get/default.tar.gz` (1 errors)
    - [ ] Unknown keys in grammar: `source.systemverilog` (in `Clams-sublimesystemverilog-cd79f5f9ca49/SystemVerilog.tmLanguage`) contains invalid keys (`hidden`)

- [ ] repository `vendor/grammars/Lean.tmbundle` (from https://github.com/leanprover/Lean.tmbundle) (1 errors)
    - [ ] Unknown keys in grammar: `source.lean` (in `Syntaxes/Lean.tmLanguage`) contains invalid keys (`Patterns[3].pattern`)

- [ ] repository `vendor/grammars/MagicPython` (from https://github.com/MagicStack/MagicPython) (1 errors)
    - [ ] Unknown keys in grammar: `source.python` (in `grammars/MagicPython.tmLanguage`) contains invalid keys (`first_line_match`)

- [ ] repository `vendor/grammars/NimLime` (from https://github.com/Varriount/NimLime) (2 errors)
    - [ ] Missing include in grammar: `source.nim` (in `Syntaxes/Nim.tmLanguage`) attempts to include `source.asm` but the scope cannot be found
    - [ ] Missing include in grammar: `source.nim` (in `Syntaxes/Nim.tmLanguage`) attempts to include `text.html.markdown.multimarkdown` but the scope cannot be found

- [ ] repository `vendor/grammars/Scalate.tmbundle` (from https://github.com/scalate/Scalate.tmbundle) (1 errors)
    - [ ] Missing include in grammar: `source.scaml` (in `Syntaxes/Scaml.tmLanguage`) attempts to include `source.js.jquery` but the scope cannot be found

- [ ] repository `vendor/grammars/Sublime-SQF-Language` (from https://github.com/JonBons/Sublime-SQF-Language) (1 errors)
    - [ ] Invalid regex in grammar: `source.sqf` (in `SQF.tmLanguage`) contains a malformed regex (regex "`\b(?i:abs|accTime|acos|action|ac`...": definition too long (52157 bytes))

- [ ] repository `vendor/grammars/atom-language-perl6` (from https://github.com/perl6/atom-language-perl6) (6 errors)
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`(?x) ( [\p{Digit}\p{Alpha}'\-_]+`...": unknown property name after \P or \p (at offset 16))
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`[\p{Digit}\p{Alpha}'\-_]+`": unknown property name after \P or \p (at offset 9))
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`(?x)(\$|@|%|&)(\.|\*|:|!|\^|~|`...": unknown property name after \P or \p (at offset 57))
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`(?x)(?<!\\)(\$|@|%|&)(?!\$)(`...": unknown property name after \P or \p (at offset 75))
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`(`": missing ) (at offset 1))
    - [ ] Invalid regex in grammar: `source.perl6fe` (in `grammars/perl6fe.cson`) contains a malformed regex (regex "`)`": unmatched parentheses (at offset 0))

- [ ] repository `vendor/grammars/ballerina-grammar` (from https://github.com/ballerina-platform/ballerina-grammar) (1 errors)
    - [ ] Unknown keys in grammar: `source.ballerina` (in `syntaxes/ballerina.tmLanguage`) contains invalid keys (`tmlVariables`, `monarchVariables`)

- [ ] repository `vendor/grammars/dartlang` (from https://github.com/dart-atom/dartlang) (1 errors)
    - [ ] Missing include in grammar: `source.yaml-ext` (in `grammars/yaml-ext.cson`) attempts to include `source.ruby.rails` but the scope cannot be found

- [ ] repository `vendor/grammars/gap-tmbundle` (from https://github.com/dhowden/gap-tmbundle) (3 errors)
    - [ ] Invalid regex in grammar: `source.gap` (in `Syntaxes/GAP.tmLanguage`) contains a malformed regex (regex "`\b(16Bits_AssocWord|16Bits_Depth`...": definition too long (65523 bytes))
    - [ ] Invalid regex in grammar: `source.gap` (in `Syntaxes/GAP.tmLanguage`) contains a malformed regex (regex "`\b(IndicesChiefNormalSteps|Indic`...": definition too long (65529 bytes))
    - [ ] Invalid regex in grammar: `source.gap` (in `Syntaxes/GAP.tmLanguage`) contains a malformed regex (regex "`\b(SMTX_GoodElementGModule|SMTX_`...": definition too long (42470 bytes))

- [ ] repository `vendor/grammars/haxe-TmLanguage` (from https://github.com/vshaxe/haxe-TmLanguage) (1 errors)
    - [ ] Missing include in grammar: `documentation.markdown.injection.haxe` (in `helper/haxe-markdown-docs.tmLanguage`) attempts to include `text.html.markdown` but the scope cannot be found

- [ ] repository `vendor/grammars/language-crystal` (from https://github.com/atom-crystal/language-crystal) (1 errors)
    - [ ] Missing include in grammar: `source.crystal` (in `grammars/crystal.cson`) attempts to include `source.js.jquery` but the scope cannot be found

- [ ] repository `vendor/grammars/language-haml` (from https://github.com/ezekg/language-haml) (1 errors)
    - [ ] Missing include in grammar: `text.haml` (in `grammars/ruby haml.cson`) attempts to include `source.ruby.rails` but the scope cannot be found

- [ ] repository `vendor/grammars/language-jsonnet` (from https://github.com/google/language-jsonnet) (1 errors)
    - [ ] Invalid regex in grammar: `source.jsonnet` (in `grammars/jsonnet.cson`) contains a malformed regex (regex "`\b[a-zA-Z_][a-z0-9A-Z_]*\s*(\([^`...": nothing to repeat (at offset 43))

- [ ] repository `vendor/grammars/language-renpy` (from https://github.com/williamd1k0/language-renpy.git) (1 errors)
    - [ ] Missing include in grammar: `source.renpy` (in `grammars/renpy.cson`) attempts to include `source.regexp.python.renpy` but the scope cannot be found

- [ ] repository `vendor/grammars/language-ruby` (from https://github.com/atom/language-ruby) (1 errors)
    - [ ] Missing include in grammar: `source.ruby` (in `grammars/ruby.cson`) attempts to include `source.js.jquery` but the scope cannot be found

- [ ] repository `vendor/grammars/language-shellscript` (from https://github.com/atom/language-shellscript) (1 errors)
    - [ ] Missing include in grammar: `source.shell` (in `grammars/shell-unix-bash.cson`) attempts to include `text.html.textile` but the scope cannot be found

- [ ] repository `vendor/grammars/language-yaml` (from https://github.com/atom/language-yaml) (1 errors)
    - [ ] Missing include in grammar: `source.yaml` (in `grammars/yaml.cson`) attempts to include `source.ruby.rails` but the scope cannot be found

- [ ] repository `vendor/grammars/liquid.tmbundle` (from https://github.com/bastilian/validcode-textmate-bundles) (1 errors)
    - [ ] Missing include in grammar: `text.html.liquid` (in `Liquid.tmbundle/Syntaxes/HTML Liquid.plist`) attempts to include `source.ruby.rails` but the scope cannot be found

- [ ] repository `vendor/grammars/mako-tmbundle` (from https://github.com/marconi/mako-tmbundle) (1 errors)
    - [ ] Missing include in grammar: `text.html.mako` (in `Syntaxes/HTML (Mako).tmLanguage`) attempts to include `comment.block` but the scope cannot be found

- [ ] repository `vendor/grammars/mathematica-tmbundle` (from https://github.com/shadanan/mathematica-tmbundle) (1 errors)
    - [ ] Invalid regex in grammar: `source.mathematica` (in `Syntaxes/Mathematica.tmLanguage`) contains a malformed regex (regex "`(\b|(?<=_))(Abort|AbortKernels|A`...": definition too long (54020 bytes))

- [ ] repository `vendor/grammars/ruby-slim.tmbundle` (from https://github.com/slim-template/ruby-slim.tmbundle) (1 errors)
    - [ ] Missing include in grammar: `text.slim` (in `Syntaxes/Ruby Slim.tmLanguage`) attempts to include `source.ruby.rails` but the scope cannot be found

- [ ] repository `vendor/grammars/sublime-autoit` (from https://github.com/AutoIt/SublimeAutoItScript) (2 errors)
    - [ ] Invalid regex in grammar: `source.autoit` (in `AutoIt.tmLanguage`) contains a malformed regex (regex "`\b(?i:_array1dtohistogram|_array`...": definition too long (39591 bytes))
    - [ ] Invalid regex in grammar: `source.autoit` (in `AutoIt.tmLanguage`) contains a malformed regex (regex "`\b(?i:_guictrltoolbar_getbuttoni`...": definition too long (39600 bytes))

- [ ] repository `vendor/grammars/turtle.tmbundle` (from https://github.com/peta/turtle.tmbundle) (5 errors)
    - [ ] Invalid regex in grammar: `source.turtle` (in `Syntaxes/Turtle.tmLanguage`) contains a malformed regex (regex "`(?x)			(?<PN_CHARS_U>[\p{L}\p{M`...": PCRE does not support \L, \l, \N{name}, \U, or \u (at offset 73))
    - [ ] Invalid regex in grammar: `source.turtle` (in `Syntaxes/Turtle.tmLanguage`) contains a malformed regex (regex "`(?x)(				(?: [\p{L}\p{M}] | [:0`...": PCRE does not support \L, \l, \N{name}, \U, or \u (at offset 121))
    - [ ] Invalid regex in grammar: `source.turtle` (in `Syntaxes/Turtle.tmLanguage`) contains a malformed regex (regex "`\[[\u20\u9\uD\uA]*\]`": PCRE does not support \L, \l, \N{name}, \U, or \u (at offset 4))
    - [ ] Invalid regex in grammar: `source.turtle` (in `Syntaxes/Turtle.tmLanguage`) contains a malformed regex (regex "`(?x)				(?<PNAME_NS> (?: (?: [\`...": PCRE does not support \L, \l, \N{name}, \U, or \u (at offset 68))
    - [ ] Invalid regex in grammar: `source.turtle` (in `Syntaxes/Turtle.tmLanguage`) contains a malformed regex (regex "`(?x)((?<=\s|^|_)(?:[\p{L}\p{M}] `...": PCRE does not support \L, \l, \N{name}, \U, or \u (at offset 57))

the grammar library contains 37 errors
```

Reply:
On `master` I see 36 errors, the new one on this branch being:

```
- [ ] repository `vendor/grammars/Sublime-SQF-Language` (from https://github.com/JonBons/Sublime-SQF-Language) (1 errors)
    - [ ] Invalid regex in grammar: `source.sqf` (in `SQF.tmLanguage`) contains a malformed regex (regex "`\b(?i:abs|accTime|acos|action|ac`...": definition too long (52157 bytes))
```